src/services/users_service.ts

```diff
@@ -1,9 +1,25 @@
 import crypto from 'crypto';
-// import { User } from '../models';
+import db from '../models';
```

Comment:
```suggestion
import model from '../models';
```
이거 model로 통일하던지 아니면 db로 통일하던지 해야할 듯...?